src/Type/ArrayType/ArrayType.php

```diff
@@ -22,10 +22,11 @@ class ArrayType extends \Consistence\ObjectPrototype
 	 * @param mixed $needle
 	 * @param boolean $strict
 	 * @return boolean
+	 * @deprecated
```

Comment:
@deprecated should be above the type annotations, see https://github.com/consistence/coding-standard/blob/master/consistence-coding-standard.md#phpdoc